darts/models/forecasting/tcn_model.py

```diff
@@ -29,7 +29,7 @@ def __init__(
         num_filters: int,
         kernel_size: int,
         dilation_base: int,
-        dropout_fn,
+        dropout: float,
```

Comment:
This is a breaking change for users that were passing a function to this argument.

Reply:
This is a hidden module so I wouldn't call it a breaking change.